lib/inspec/resources/virtualization.rb

```diff
@@ -253,7 +304,40 @@ def collect_data_linux
       return if detect_openvz
       return if detect_openstack
       return if detect_parallels
-      # TODO: where is vmware?
+      return if detect_vmware
+      return if detect_hyperv
+    end
+
+    def windows_computer_system
+      script = "Get-WmiObject -Class Win32_ComputerSystem -ErrorAction Stop | ConvertTo-Json"
+      cmd = inspec.powershell(script)
+
+      JSON.parse(cmd.stdout)
+    rescue JSON::ParserError => _e
+      nil
+    end
+
+    def collect_data_windows
```

Comment:
I believe this will cause the system to return true for the `physical_system?` method if it is virtualised but is not in the list of hypervisors we check for below.
I think this is acceptable, but we might have to help others in future who ask why InSpec tells them their VM is physical.

Reply:
I see this is now in the documentation, thanks!